src/utils.js

```diff
@@ -0,0 +1,41 @@
+export function getStyleFromGoogleVariant( variant ) {
+	return variant.includes( 'italic' ) ? 'italic' : 'normal';
+}
+
+export function getWeightFromGoogleVariant( variant ) {
+	return variant === 'regular' || variant === 'italic'
+		? '400'
+		: variant.replace( 'italic', '' );
+}
+
+export function getGoogleVariantFromStyleAndWeight( style, weight ) {
+	if ( weight === '400' ) {
+		if ( style === 'italic' ) {
+			return 'italic';
+		}
+		return 'regular';
+	}
+	if ( style === 'normal' ) {
+		return weight;
+	}
+	return weight + style;
+}
+
+export function forceHttps( url ) {
+	return url.replace( 'http://', 'https://' );
+}
+
+export function bytesToSize( bytes ) {
+	const sizes = [ 'Bytes', 'KB', 'MB', 'GB', 'TB' ];
+	if ( bytes === 0 ) return 'n/a';
+	const i = parseInt( Math.floor( Math.log( bytes ) / Math.log( 1024 ) ) );
+	if ( i === 0 ) return bytes + ' ' + sizes[ i ];
+	return ( bytes / Math.pow( 1024, i ) ).toFixed( 1 ) + ' ' + sizes[ i ];
+}
```

Comment:
Let's use the package [pretty-bytes](https://www.npmjs.com/package/pretty-bytes). It also supports locale.

Reply:
Does the locale add anything apart from changing a `.` for a `.` in some locales? 
If it is just that seems like an overkill to add a library to replace a 5-line function.